src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -120,7 +120,10 @@ class DiskImageEditor extends Component {
           bootable: disk.get('bootable'),
           size: 0,
           storageDomain: disk.get('storageDomainId'),
-          format: disk.get('format'),
+
+          // NOTE: webadmin always uses raw when creating disks.  We need to look at
```

Comment:
This is no so accurate (check https://github.com/oVirt/ovirt-web-ui/pull/1218#issuecomment-638751656 #). It can be either raw or cow.